package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "com.unity.cinemachine",
     "displayName": "Cinemachine",
-    "version": "2.7.2",
+    "version": "2.8.0",
```

Comment:
Should be 2.8.0-pre.1.
Here only.  The other places are fine.
Also: add an entry to changelog.md to start the new section